src/components/panel/panel.scss

```diff
@@ -28,41 +28,43 @@
 .md-panel {
   opacity: 0;
   overflow: auto;
+  position: fixed;
 
   &._md-panel-shown {
     // Only used when custom animations are present.
     // Overridden by the default animations.
     opacity: 1;
     transition: none;
+  }
 
-    &._md-panel-animate-slide-in {
-      opacity: 1;
-      transition: $swift-ease-out, opacity 1ms;
-    }
-
-    &._md-panel-animate-scale-in {
-      opacity: 1;
-      transition: $swift-ease-out;
-    }
-
-    &._md-panel-animate-fade-in {
-      opacity: 1;
-      transition: all 0.4s linear;
-    }
+  &._md-panel-animate-enter {
+    opacity: 1;
+    transition: all 0.5s cubic-bezier(0.0, 0.0, 0.2, 1);
```

Comment:
Can you add variables for these transitions? It looks like you use 2 different ones in this file.